dev-python/h5py/h5py-2.10.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1 flag-o-matic
+
+DESCRIPTION="Simple Python interface to HDF5 files"
+HOMEPAGE="https://github.com/h5py/h5py https://www.h5py.org"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="doc test examples mpi"
```

Comment:
Please sort ... and `distutils_enable_tests` should add `IUSE+=" test"

Reply:
Repoman seems to want it in this order, any other order will output:
```
RepoMan scours the neighborhood...
KEYWORDS.unsorted             1
dev-python/h5py/h5py-2.10.0.ebuild contains unsorted keywords
```

Reply:
but I was talking about the IUSE list ;)

Reply:
Oops, my bad, well removing `IUSE=test` automatically made it sorted anyway :D